charts/chronograf/values.yaml

```diff
@@ -95,6 +98,20 @@ oauth:
     client_secret: CHANGE_ME
     # This is a comma separated list of Heroku organizations (OPTIONAL)
     he_orgs: ""
+  generic:
+    enabled: false
+    client_id: CHANGE_ME
```

Comment:
This should be commented out, which should cause the chart to fail if the user fails to provide the values

Reply:
This goes for all the "CHANGE_ME"'s

Reply:
I commented out the "CHANGE_ME"'s, rebased the branch and squashed one of the commits.